CHANGELOG.md

```diff
@@ -5,6 +5,34 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### [1.0.16] - 2025-01-17
+
+### Added
+
+- defaultLang support value can be set in Sections BO: Read the new default language field from project metadata #117
+
+- Sections Tutorial: Interactive user guides #115
+
+- Translation management for configurable fields #109
+
+### Updated
```

Comment:
please indicate the performance improvements you have done so far

Reply:
Done